composer.json

```diff
@@ -1,34 +1,35 @@
-{
-    "name"              : "hostnet/webpack-bundle",
-    "description"       : "Integrates Webpack with Symfony2",
-    "authors"           : [{ "name": "Harold Iedema", "email" : "hiedema@hostnet.nl" }],
-    "license"           : "MIT",
-    "type"              : "symfony-bundle",
-    "keywords"          : ["symfony", "webpack", "bundle", "asset", "resources", "javascript", "css", "less", "content"],
-    "require"           : {
-        "php"                          : "^5.5|^7.0",
-        "symfony/symfony"              : "^2.7|^3.0",
-        "symfony/monolog-bundle"       : "^2.7|^3.0",
-        "twig/twig"                    : "^1.18|^2.0"
-    },
-    "require-dev" : {
-        "phpunit/phpunit" : "^4.7|^5.0",
-        "symfony/phpunit-bridge": "^2.7|^3.0"
-    },
-    "autoload" : {
-        "psr-4" : {
-            "Hostnet\\Bundle\\WebpackBundle\\" : "src/Bundle",
-            "Hostnet\\Component\\Webpack\\"    : "src/Component"
-        }
-    },
-    "autoload-dev" : {
-        "psr-4" : {
-            "Hostnet\\Fixture\\WebpackBundle\\" : "test/Fixture",
-            "Hostnet\\Bundle\\WebpackBundle\\"  : "test/Bundle",
-            "Hostnet\\Component\\Webpack\\"     : "test/Component"
-        }
-    },
-    "archive": {
-        "exclude": [ "/test" ]
-    }
-}
+{
+    "name":        "hostnet/webpack-bundle",
+    "type":        "symfony-bundle",
+    "description": "Integrates Webpack with Symfony2",
+    "license":     "MIT",
+    "require": {
+        "php":                    "^7.0",
+        "symfony/monolog-bundle": "^3.1.0",
+        "symfony/symfony":        "3.3.*,>=3.3.2",
+        "twig/twig":              "^2.4.3"
+    },
+    "require-dev": {
+        "hostnet/phpcs-tool":     "^4.1.4",
```

Comment:
Rule exclusion doesn't seem to work properly / `phpcs.xml.dist` isn't read from the repository root.